flink-kubernetes-operator/pom.xml

```diff
@@ -208,6 +208,18 @@ under the License.
             <artifactId>junit-jupiter-params</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.javaoperatorsdk</groupId>
+            <artifactId>kubernetes-webhooks-framework-core</artifactId>
+            <version>${operator.sdk.webhook-framework.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Why do we need this new dependency inside the operator?

Reply:
This  dependency is for the Kubernetes-webhooks-framework. This can be removed as we don't have any strong relation with this dependency.